Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1306,6 +1306,14 @@ namespace ScriptCanvasEditor
             return;
         }
 
+        // Redirect script event file open process
+        AZ::IO::Path filePath = fullPath;
+        if (filePath.Extension() == ".scriptevents")
+        {
+            OnScriptEventOpen(fullPath);
+            return;
+        }
+
```

Comment:
Before a change was made, Script Canvas functions were handled as separate assets. This had the benefit of a separation of concerns, it wasn't necessary to check by file extension. Maybe as part of a second iteration we can work towards bringing back that ability to make Script Canvas more versatile in the asset types it can edit.